applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_server.c

```diff
@@ -546,8 +534,9 @@ int unicast_server_send(struct encoded_audio enc_audio)
 	struct net_buf *buf;
 	static bool hci_wrn_printed;
 
-	if (enc_audio.num_ch != 1) {
-		LOG_ERR("Num encoded channels must be 1");
+	if (enc_audio.num_ch > CONFIG_BT_ASCS_ASE_SRC_COUNT) {
```

Comment:
Hmm the rest of the send function does currently only handle one stream. Should we fix this before changing to CONFIG_BT_ASCS_ASE_SRC_COUNT?

Reply:
We could add a compile time error that fails if CONFIG_BT_ASCS_ASE_SRC_COUNT != 1